gtsam/symbolic/SymbolicConditional.h

```diff
@@ -17,10 +17,10 @@
 
 #pragma once
 
-#include <gtsam/symbolic/SymbolicFactor.h>
-#include <gtsam/inference/Conditional.h>
 #include <gtsam/base/Testable.h>
 #include <gtsam/base/types.h>
+#include <gtsam/inference/Conditional-inst.h>
```

Comment:
`inst` headers should not be included?

Reply:
Had linking errors that I could not resolve for the life of me.